audio_common_msgs/msg/AudioInfo.msg

```diff
@@ -0,0 +1,5 @@
+uint8 channels       # Number of channels
+uint32 sample_rate   # Sampling rate [Hz]
+string sample_format # Audio format (e.g. S16LE)
+uint32 bitrate       # Amount of audio data per second [bits/s]
+string coding_format # Audio coding format (e.g. WAVE, MP3)
```

Comment:
can you move comment before field like below?
```
# Number of channels
uint8 channels
# Sampling rate [Hz]
uint32 sample_rate
...
...
```

you can see an example for `moveit_msgs`.
https://github.com/ros-planning/moveit_msgs/blob/master/msg/MotionPlanRequest.msg